src/main_dpdk.cpp

```diff
@@ -1955,7 +2009,11 @@ CCoreEthIFStateless::send_node_packet(CGenNodeStateless      *node_sl,
         }
         return send_node_flow_stat(m, node_sl, lp_port, lp_stats, (node_sl->get_cache_mbuf()) ? true : false);
     } else {
-        return send_pkt(lp_port, m, lp_stats);
+        if(unlikely(node_sl->should_send_immediately())) {
```

Comment:
flow_stats is always immediately

Reply:
Thanks

Reply:
Hi @hhaim, thanks you for review!
As far as I understand, this part of code is referring to any other nodes as check of ->is_stat_needed() is above it (1950/2004). As normally if node don't has stat_needed it will go buffered path.
This part is trying to allow normal node with additional flag to be send immediately without buffering.